### PR TITLE
Don't send scriptParsedEvents with skipList support

### PR DESCRIPTION
**Pull request by grouma**
We no longer need to black box the Dart SDK as we now leverage Chrome's skipList which essentially serve the same purpose. There is a very minor stepping performance regression as we don't send all cached skipLists but I intend to fix this in a follow up PR.

The blackbox logic was the remaining bit of logic that required the script parsed events. With that gone, we can now ignore the events entirely.

### CODE REVIEW

File: dwds/debug_extension/web/background.dart [hunk 1]
```diff
@@ -302,15 +288,11 @@ class _EventQueue {
...
     var event = _extensionEventFor(method, params);
-    if (_supportsBatching && method == 'Debugger.scriptParsed') {
-      if (queuedEvents.isEmpty) {
-        _startTimer();
-      }
-      queuedEvents.add(event);
-    } else {
-      _forward(event);
-    }
+    _forward(event);
```
Comment (grouma) [inline dwds/debug_extension/web/background.dart:295]: Note, we no longer need to batch events for performance reasons as we have resolved the performance issues with `package:sse`.

File: dwds/debug_extension/web/background.dart [hunk 2]
```diff
@@ -260,32 +260,18 @@ Future<void> _startSseClient(
...
-  static const _flushInterval = Duration(milliseconds: 250);
-
   final SocketClient _client;
   final Tab _currentTab;
   bool _attached;
-  bool _supportsBatching;
+  bool _supportsSkipLists;
 
   /// The pending events.
   final queuedEvents = <ExtensionEvent>[];
```
Comment (annagrin) [inline dwds/debug_extension/web/background.dart:273]: looks like queuedEvents are no longer used - should it be removed?

File: dwds/debug_extension/web/background.dart [hunk 3]
```diff
@@ -302,15 +288,11 @@ class _EventQueue {
     if (source.tabId != _currentTab.id || !_attached) {
```
Comment (annagrin) [inline dwds/debug_extension/web/background.dart:288]: Since this method is no longer enqueueing, should the name change?

**Review by annagrin — APPROVED** [review]
LGTM (with comments)

File: dwds/debug_extension/web/background.dart (hunk 3 above)
Comment (grouma) [reply dwds/debug_extension/web/background.dart:285]: Done. Updated the comment too.

File: dwds/debug_extension/web/background.dart (hunk 2 above)
Comment (grouma) [reply dwds/debug_extension/web/background.dart]: Done.

**Review by annagrin — APPROVED** [review]
LGTM!

**Pull request merged**